charts/datadog/values.yaml

```diff
@@ -254,6 +254,19 @@ datadog:
   ## ref: https://docs.datadoghq.com/getting_started/tagging/assigning_tags/?tab=containerizedenvironments#environment-variables
   checksCardinality:  # low, orchestrator or high (not set by default to avoid overriding existing DD_CHECKS_TAG_CARDINALITY configurations, the default value in the Agent is low)
 
+  # SBOM configuration
+  sbom:
+    # datadog.sbom.enabled -- Enable SBOM (Software Bill Of Materials) collection
+    enabled: false
+
+    containerImage:
```

Comment:
I'm not sure we want to attached the `container-image` collection under the `sbom` section